src/screens/ProfileScreen.tsx

```diff
@@ -127,12 +128,12 @@ const Field = ({ label, value }: FieldProps) => {
 
   return (
     <View style={styles.fieldView} testID={tID(label)}>
-      <Text variant="labelMedium" style={styles.fieldLabelText}>
-        {label}
-      </Text>
-      <Text variant="bodyLarge" style={styles.fieldValueText}>
-        {value}
-      </Text>
+      <Text style={styles.fieldLabelText}>{label}</Text>
+      <Text style={styles.fieldValueText}>{value}</Text>
+      <Divider
+        style={styles.divider}
+        theme={{ colors: { outlineVariant: '#727983' } }}
```

Comment:
Could consider pulling this theme object out of the component

Reply:
I think the divider should take the color directly from the theme that comes from the root theme provider.